content/en/blog/_posts/2020-04-06-API-Priority-and-Fairness-Alpha.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog
+title: "API Priority and Fairness Alpha"
+date: 2020-04-06
+slug: kubernetes-1-18-feature-api-priority-and-fairness-alpha
+---
+
+**Authors:** Min Kim (Ant Financial), Mike Spreitzer (IBM), Daniel Smith (Google)
+
+This blog describes “API Priority And Fairness”, a new alpha feature in Kubernetes 1.18. API Priority And Fairness permits cluster administrators to divide the concurrency of the control plane into different weighted priority levels. Every request arriving to kube-apiserver will be categorized into one of the priority levels and get its fair share of the control plane’s throughput.
+
+## What problem does this solve?
+Today the apiserver has a simple mechanism for protecting itself against CPU and memory overloads: max-in-flight limits for mutating and for read-only requests. Apart from the distinction between mutating and readonly, no other distinctions are made among requests; consequently, there can be undesirable scenarios where one subset of the requests crowds out other requests.
+
+In short, it is far too easy for Kubernetes workloads to accidentally DoS the apiservers, causing other important traffic--like system controllers or leader elections---to fail intermittently. In the worst cases, a few broken nodes or controllers can push a busy cluster over the edge, turning a local problem into a control plane outage.
+
+## How do we solve the problem?
+The new feature “API Priority and Fairness” is about generalizing the existing max-in-flight request handler in apiserver, to make the behavior more intelligent and configurable. The overall approach is as follows.
```

Comment:
Let's make it plural "apiservers".  Alternatives include making it "each apiserver", or "the apiservers".